src/mocks/handlers.ts

```diff
@@ -1,26 +1,30 @@
-import { rest } from 'msw'
+import { http, HttpResponse } from 'msw'
 import { myProfile, memberProfileList, articles } from './fixture'
-import type { Articles, MemberProfile, MyProfile } from '@types'
+import type { Articles, MemberProfile, MyProfile, TradeStatus } from '@types'
 
 export const handlers = [
   /**
    * 내 사용자 프로필 정보 조회
    */
-  rest.get('/members/mypage', (_, res, ctx) => {
-    return res(ctx.status(200), ctx.json<MyProfile>(myProfile))
+  http.get('/members/mypage', () => {
+    return HttpResponse.json<MyProfile>(myProfile, { status: 200 })
+  }),
+
+  http.get('/test', async () => {
+    return HttpResponse.json<MyProfile>(myProfile, { status: 200 })
   }),
 
   /**
    * 타 사용자 프로필 정보 조회
    * members/{member-id}
    * 현재는 member-id 1과 2만 존재
    */
-  rest.get('/members/:memberId', (req, res, ctx) => {
-    const { memberId } = req.params
-    return res(
-      ctx.status(200),
-      ctx.json<MemberProfile>(memberProfileList[memberId as string])
-    )
+  http.get('/members/:memberId', ({ params }) => {
+    const id = params.memberId as string
```

Comment:
memberId 타입이 number라서 key로 사용하지 못하니 as 사용보다 string으로 변경해주면 될 것 같아요 !

```suggestion
    const id = String(params.memberId)
```

Reply:
[오나료](healt check를 위해 넣어놨던건데 뺴겠습니다~)